recipes/r-rgeos/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.3-26' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rgeos
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rgeos_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rgeos_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rgeos/rgeos_{{ version }}.tar.gz
+  sha256: 98524a0b8113abe6c3d0ecc1f2f66e7ab6d40c783a76158cfc017e1ab1e3f433
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-sp >=1.1_0
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-sp >=1.1_0
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('rgeos')"  # [not win]
+    - "\"%R%\" -e \"library('rgeos')\""  # [win]
+
+about:
+  home: https://r-forge.r-project.org/projects/rgeos/ http://trac.osgeo.org/geos/
```

Comment:
The CRAN entry has two URLs. I've never encountered this before. Presumably conda will still build the package fine, but Anaconda Cloud may not be sure how to display this.